plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/clients/consumer/PlainTextKafkaConsumerFactory.java

```diff
@@ -57,6 +61,7 @@ public Properties configure(ConnectorSession session)
         properties.setProperty(VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
         properties.setProperty(RECEIVE_BUFFER_CONFIG, Long.toString(kafkaBufferSize.toBytes()));
         properties.setProperty(ENABLE_AUTO_COMMIT_CONFIG, Boolean.toString(false));
+        properties.setProperty("security.protocol", securityProtocol.getName());
```

Comment:
static import `org.apache.kafka.clients.CommonClientConfigs#SECURITY_PROTOCOL_CONFIG` and use instead.
(also update other occurences)